### PR TITLE
add link to babel-plugins repo and npm

### PR DESCRIPTION
**Pull request by hzoo**

### CODE REVIEW

File: docs/advanced/plugins/index.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ redirect_from:
   </p>
 </blockquote>
 
+You can also find official plugins at https://github.com/babel-plugins and other community plugins by searching [npm](https://www.npmjs.com/search?q=babel-plugin-).
```
Comment (thejameskyle) [inline docs/advanced/plugins/index.md]: mind putting this in a callout like the items above?

Comment (hzoo) [inline docs/advanced/plugins/index.md]: k (Also I wonder if github will ever support modifying a PR like this)

Comment (thejameskyle) [inline docs/advanced/plugins/index.md]: Hmm? You can continue pushing to this branch

Comment (hzoo) [inline docs/advanced/plugins/index.md]: Sorry I meant from the website (I just edited the file on the website which made the `patch-1` branch)

Comment (thejameskyle) [inline docs/advanced/plugins/index.md]: screen shot 2015-07-01 at 7 07 34 pm

You can click the edit button on the top right of the file in the diff.

Comment (hzoo) [inline docs/advanced/plugins/index.md]: Ah ok cool it does commit to the same branch! I guess the only issue is you can't squash into a single commit without doing that locally and force pushing

**Comment by thejameskyle** [comment]
noice thanks

**Pull request merged**